src/appfl/run_mpi.py

```diff
@@ -75,10 +78,15 @@ def run_server(
     else:
         cfg.validation = False
 
+    # Broadcast limitation according to the procs numbers
+    intmax = 2100000000 # INT_MAX,2147483647, does not fit on the MPI send buffer size
+    recvlimit = math.floor(intmax/(comm_size-1)) # The total recv size should be less than intmax due to the MPI recv buffer size
```

Comment:
Out of curiosity, is there any document for this? How this number 2100000000 is obtained, and why recvlimit is intmax/commsize-1

Reply:
@Zilinghan good point!
@yim0331 : I think the size may be dependent to the integer type too: `int8`, `int16`, `int32`, ...